src/components/dialogs/line-split-with-voltage-level-dialog.js

```diff
@@ -180,6 +181,7 @@ const LineSplitWithVoltageLevelDialog = ({
                     : '',
         });
     const voltageLevelOrIdRef = useRef();
+    voltageLevelOrIdRef.current = voltageLevelOrId;
```

Comment:
Could use 
`const voltageLevelOrIdRef = useRef(voltageLevelOrId);`
instead of 
```
const voltageLevelOrIdRef = useRef();
voltageLevelOrIdRef.current = voltageLevelOrId;
```